content/dstu2/immunization.md

```diff
@@ -20,9 +20,12 @@ Search for Immunizations that meet supplied query parameters:
 ### Parameters
 
  Name      | Required? | Type          | Description
------------|-----------|---------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-----------|-----------|---------------|-----------------------------------------------------------------------------------------------------
  `patient` | Y         | [`reference`] | The patient for the vaccination record. Example: `12345`
- `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'. Single dates are treated as implied ranges. EG: `date=ge2014-09-24` `&date=le2015-09-24`
+ `date`    | N         | [`date`]      | Date range into which the immunization administration date falls. Must be prefixed by 'ge' or 'le'.
+
+Notes:
+ - The date parameter value should be prefixed once by 'ge' representing the earliest date, and once by 'le' representing the latest date. Examples: 'date=ge2015-01-01&date=le2016-01-01', 'date=ge2015-01-01', 'date=le2016-01-01'
```

Comment:
Added ticks around parameter name and the parameter examples with https://github.com/cerner/fhir.cerner.com/pull/93/commits/736a70bf0d1091772672451f354938a173ebcb25